servlet/src/jettyTest/java/io/grpc/servlet/GrpcServletSmokeTest.java

```diff
@@ -99,7 +99,7 @@ public void tearDown() {
   @Test
   public void unaryCall() {
     Channel channel = cleanupRule.register(
-        ManagedChannelBuilder.forAddress(HOST, port).usePlaintext().build());
+        NettyChannelBuilder.forAddress(HOST, port).usePlaintext().build());
```

Comment:
This should have been fine already. The point wasn't to avoid ManagedChannelBuilder. The point was to avoid the wrong-assumption casts.

Reply:
Undid that one. Yes I was just going through the directory rather than waiting for them to fail one by one.